src/EntityGraphQL/Compiler/GqlNodes/ExecutableGraphQLStatement.cs

```diff
@@ -82,6 +82,7 @@ QueryRequestContext requestContext
         // }
         // people & movies will be the 2 fields that will be 2 separate expressions
         var result = new ConcurrentDictionary<string, object?>();
+        var validator = new GraphQLValidator();
```

Comment:
We should get it from the service provider. Allowing people to implement their own. But we only want to fetch it from the service provider once per mutation field - likely per top level field.